# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kahakai/glibc23.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

errno.h needs to be included explicitly ...

  - Rene Rebe <rene@exactcode.de>

--- ./src/Screen.cc.vanilla	2004-11-06 17:13:12.615169843 +0100
+++ ./src/Screen.cc	2004-11-06 17:13:40.718495227 +0100
@@ -59,6 +59,8 @@
 #endif // HAVE_SIGNAL_H
 }
 
+#include <errno.h>
+
 #include <algorithm>
 #include <sstream>
 #include <iostream>
